src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+const ADDED = 'app/reducer/ADDED';
+const REMOVED = 'app/reducer/REMOVED';
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADDED:
+      return [
+        ...state,
+        {
+          author: action.author,
+        },
+      ];
+    case REMOVED:
+      return state.splice(action.id, 1);
+    default: return state;
+  }
+};
+
+export function addBook() { return { type: ADDED }; }
+export function removeBook() { return { type: REMOVED }; }
+export default bookReducer;
```

Comment:
- [ ] Good job so far, but I can't add or remove a book to your bookstore, please kindly fix that 🙃 🙃